cmd/src/actions_exec.go

```diff
@@ -182,15 +177,23 @@ Format of the action JSON files:
 
 		ctx := context.Background()
 
+		logger := newActionLogger(*verbose, *keepLogsFlag)
+
 		diffSupportsNoDereference, err = diffSupportsFlag(ctx, "--no-dereference")
 		if err != nil {
 			return err
 		}
+		if !diffSupportsNoDereference {
+			logger.Warnf("Local installation of 'diff' doesn't support '%s' flag. Consider upgrading to GNU diff >=3.7.\n", "--no-dereference")
```

Comment:
3.3 I think. For some reason the `suggestion` button is disabled so I cannot make one :facepalm:

Reply:
You think this is necessary? Newest version is `3.7`, I think we should recommend upgrading to that :)

Reply:
yup definitely makes sense to have people use always the most recent version, I was thinking it might confuse people when they are on some OS that doesn't have 3.7 in their package lists that they cannot use src-cli, but it might be unreasonable to account for that 😁

Reply:
Let's stay with this for now, since I also want to explore some other options (to fix #137) and this might be invalidated soon.

Reply:
sure :)